plugins/link/plugin.js

```diff
@@ -873,5 +880,25 @@
 		 * @cfg {Boolean} [linkJavaScriptLinksAllowed=false]
 		 * @member CKEDITOR.config
 		 */
+
+		/**
+		 * Optional JavaScript Regular Expression whenever telephone numbers in Link dialog should be validated.
+		 *
+		 *		config.linkTelNumberValidate_regExp = /^[0-9]{9}$/;
+		 *
+		 * @since 4.11.0
+		 * @cfg {RegExp} [linkTelNumberValidate_regExp]
```

Comment:
I'm thinking about changing names of variables to `linkTelNumberRegExp` and `linkTelNumberMsg`. ATM only these two config variables in link plugin contain underscore, introducing some kind of inconsistency.